Competition2018/src/main/java/competition/subsystems/climberdeploy/ClimberdeploySubsystem.java

```diff
@@ -1,54 +1,70 @@
 package competition.subsystems.climberdeploy;
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 
 import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
 @Singleton
 public class ClimberdeploySubsystem extends BaseSubsystem {
 	
-	DoubleProperty deploySpeed;
+	double currentDeploySpeed;
+	DoubleProperty fastDeploySpeed;
+	DoubleProperty slowDeploySpeed;
+	CommonLibFactory clf;
+	
+	public XCANTalon motor;
+
 	
 	@Inject
-	public ClimberdeploySubsystem(XPropertyManager propMan) {
-		propMan.createPersistentProperty("deploySpeed", 0.2);
+	public ClimberdeploySubsystem(CommonLibFactory clf, XPropertyManager propMan) {
+		this.clf = clf;
+		currentDeploySpeed = 0.2;
```

Comment:
instead of hard-coding this, have it be `currentDeploySpeed = fastDeploySpeed.get()`